ocs_ci/deployment/disconnected.py

```
@@ -256,31 +309,59 @@ def mirror_index_image_via_oc_mirror(index_image, packages, idms=None):
         os.makedirs(os.path.expanduser("~/.docker/"), exist_ok=True)
         os.symlink(pull_secret_path, os.path.expanduser(docker_config_file))
 
+    # Configure registries.conf if requested (for FDF internal images)
+    if configure_registries:
+        logger.info("Configuring /etc/containers/registries.conf for internal images")
+        registries_template = os.path.join(
+            constants.FDF_TEMPLATE_DIR, "registries.conf.template"
+        )
+        if os.path.exists(registries_template):
+            try:
+                with open(registries_template, "r") as f:
```

Ruff (0.15.13)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)